tests/test_nplinker_local.py

```diff
@@ -42,9 +42,28 @@ def npl() -> NPLinker:
     return npl
 
 
+# ---------------------------------------------------------------------------------------------------
+# After manually checking data files for PODP MSV000079284, we have the following numbers:
+# 370 antismash BGCs
```

Comment:
```suggestion
# 370 antismash BGCs and 20 MiBiG BGCs
```

Reply:
Better to match the numbers tested below, otherwise is confusing

Reply:
You're right. Updated the comments to make it more clear.